coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -1,47 +1,55 @@
 <template>
-  <div>
-    <ConfirmDialog ref="confirm" />
+  <div id="todo-list">
+    <ConfirmDialog
+      ref="confirm"
+      attach="#todo-list"
+    />
 
     <DeleteErrorDialog
       :dialog="deleteErrorDialog"
       :errors="deleteErrors"
       :warnings="deleteWarnings"
       @okay="resetErrors"
+      attach="#todo-list"
     />
 
     <v-expansion-panels v-if="taskItems && taskItems.length > 0" accordion>
       <v-expansion-panel
-        class="todo-list"
-        v-for="(item, index) in orderBy(taskItems, 'order')"
-        v-bind:key="index"
+        class="align-items-top todo-item"
         expand-icon=""
-        :class="{ 'disabled': !item.enabled, 'draft': isDraft(item),
-        'awaitConfirm': !confirmCheckbox && isConfirmEnabled(item.type, item.status)}">
-
-        <v-expansion-panel-header class="no-dropdown">
+        v-for="(item, index) in orderBy(taskItems, 'order')"
+        :key="index"
+        :class="{
+          'disabled': !item.enabled,
+          'draft': isDraft(item),
+          'awaitConfirm': !confirmCheckbox && isConfirmEnabled(item.type, item.status)
+        }"
+      >
+        <v-expansion-panel-header class="todo-item-toggle no-dropdown">
           <div class="list-item">
-            <div class="filing-type">
+            <div class="todo-label">
               <div class="list-item__title">{{item.title}}</div>
+
               <div class="bcorps-ar-subtitle"
-                 v-if="entityFilter(EntityTypes.BCORP) &&
-                 isConfirmEnabled(item.type, item.status)"
+                v-if="entityFilter(EntityTypes.BCORP) && isConfirmEnabled(item.type, item.status)"
```

Comment:
I like putting everything on one line as well, but sometimes i find myself breaking up the lines for consistency with other components when the properties get a little lengthy. Especially on those components that use a lot of small props built into the component. What is our rule of thumb, when to break it up into multiple lines? Use the limit set by Linting?

Reply:
Personally I don't like very long lines because I often have 2 windows side by side and I don't like scrolling horizontally to see everything :>
I also don't like _every single attribute_ on separate lines.
So, like you, I strive for consistency within a component. There are a few other things I consider, such as:
- if possible, move style attributes (such as "outlined") close to the tag, so it can be read as "outline button"
- if possible, move class close to the tag (some devs will say class should always be first, but I have lots of personal exceptions)
- if possible, move logic (v-if, v-show) close to the tag name, as it makes reviewing the code easier, especially when folding blocks
- if possible, move looping (v-for) attribute before other attributes that use the looping variables (`:class`, `:key`, `:id`, etc)

Also, you may have seen that I added spans around some standalone text. This is because most text is an inline element (vs block) and I don't want extra whitespace around it (even though whitespace is usually removed). Consider this template:
```
<v-btn>
  Click me
</v-btn>
```
which renders as:
```
<button>
  <span>
    " Click me "
  </span>
</button>
```
versus
```
<v-btn>Click me</v-btn>

<v-btn>
  <span>Click me too</span>
</v-btn>
```
which renders as:
```
<button>
  <span>
    "Click me"
  </span>
</button>

<button>
  <span>
    <span>Click me too</span>
  </span>
</button>
```
In the latter example, you can select the button text and it won't contain extra spaces.

Reply:
wow, thanks Severin! That was a really informative and helpful comment, appreciate that. I think i will extract this little bit of information and will certainly reference it when i'm faced with another 'To line break or not to line break' dilemma. Thanks!